### DISCUSSION TITLE
安装后 eslint 报错

### DISCUSSION THREAD

**Discussion by qingtian5266** [jdf2e/nutui#1753] [General]
你们安装完后不提示代码错误吗？我看了下确实是错的啊
![image](https://user-images.githubusercontent.com/28749518/195816395-5c146264-e97d-4ddc-9080-cbbf2773949c.png)

**Comment by ntnyq** [comment]
当前仓库未做严格的 ESLint 校验。lint-staged 只做了 prettier 格式化。
